### DISCUSSION TITLE
Any way to change Menu name

### DISCUSSION THREAD

**Discussion by SlavicBlin** [YimMenu/YimMenu#1364] [General]
Iam trying to figure out how to change the menu name when the notification pops up saying "welcome Loaded YimMenu"
i do see that it uses the _T for title i assume but cant find where to change the Title. any help would be great... :)

**Comment by xiaoxiao921** [comment]
Thank you for your interest in the YimMenu project and for reaching out to us. We appreciate your enthusiasm, but I wanted to clarify the purpose and scope of this repository.

Firstly, please understand that this project is a labor of love, and we dedicate our free time to its development and maintenance. Our time is limited, and we strive to focus on addressing genuine issues and pull requests that contribute to the project's growth and improvement.

While we understand your desire to make changes, it is important to maintain the integrity and purpose of the project. Personalized modifications, such as changing the project name for individual use, do not align with the intended goals of the repository.

However, if you are interested in contributing to the project or have any questions related to its intended functionality, I would be more than happy to assist you. We encourage users to participate in open-source projects and learn from the community's collective knowledge.

In the meantime, I recommend exploring online tutorials, documentation, or programming forums to gain a deeper understanding of programming concepts and practices. This will empower you to make changes to projects and develop your skills further.

Thank you for your understanding, and I hope you find the resources mentioned above helpful for your programming journey.

Best regards
